201-IoT-IotHub-EdgeEmulator-VM/azuredeploy.json

```diff
@@ -0,0 +1,275 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vmName": {
+            "type": "string",
+            "defaultValue": "EdgeVM",
+            "metadata": {
+                "description": "The name of you Virtual Machine."
+            }
+        },
+        "IoTHubname": {
+            "type": "string",
+            "defaultValue": "[concat('IoThub', resourceGroup().name)]",
```

Comment:
does this need to be globally unique?

Reply:
At the template documentation it just says:

"The name of the IoT hub to create or update." What makes me assume that it doesn't need to be globally unique, but perhaps it will be a good practice make it unique.